test/e2e/kubectl/kubectl.go

```diff
@@ -1939,6 +1939,43 @@ metadata:
 		})
 	})
 
+	ginkgo.Describe("Kubectl events", func() {
+		var podName string
+
+		ginkgo.BeforeEach(func() {
+			podName = "e2e-test-httpd-pod"
```

Comment:
Maybe we should change generic pod name to something specific to events to prevent possible conflicts with others tests?

Reply:
Each e2e create its own namespace to avoid this, you can also leverage the garbage collector to delete the test resources, the framework delete the namespace after test has finished